toolkit/scripts/toolchain.mk

```diff
@@ -197,7 +197,7 @@ $(toolchain_rpms): $(toolchain_manifest) $(toolchain_local_temp)
 else
 # Download from online package server
 $(toolchain_rpms):
-	@rpm_filename="$(notdir $@)" && \
+	rpm_filename="$(notdir $@)" && \
```

Comment:
what does removing the @ do?

Reply:
By default, makefile echoes the command it will run to the stdout. Adding `@` prevents that behavior. However, since we've added `$(toolchain_rpms)` target to `.SILENT` pseudotarget (L162), this is no longer needed.